templates/Program/program_download_disabled_button.html.twig

```diff
@@ -0,0 +1,18 @@
+<button id="projectDownloadDisabledButton{{ suffix|default('') }}" class="btn btn-primary btn-block js-btn-project-download-disabled"
+        data-redirect-url='{{ login_redirect }}'
+        data-alert-text="{{ 'downloadDisabledAlert'|trans({}, 'catroweb')}}"
+
+>
+
+  {# Loading spinner during download #}
+  <i id="projectDownloadDisabledButton__loadingSpinner{{ suffix|default('') }}" class="material-icons text-start d-none">
+    {% include 'components/loading_spinner.html.twig' with {'spinner_id': 'download-disabled-progressbar' ~ suffix|default(), 'size': 'small'} only %}
+  </i>
+
+  {# Icon #}
+  <i id="projectDownloadDisabledButton__icon{{ suffix|default('') }}" class="material-icons align-bottom">lock</i>
+
+  {# Text #}
+  <span>{{ "download"|trans({}, "catroweb") }}</span>
+
+</button>
```

Comment:
is really a new template needed to disable a button? maybe think about adding a different variable to make the existing template more flexible and reduce the duplicated code

Reply:
I was thinking about the same but if you add a new variable instead, you could go into dev tools and change it form eg 0 to 1 and by that you could enable the download button, even if you are not logged in.

Reply:
The request is locked anyway to users not logged in, so the user would not achieve anything by doing such things. However, it's also okay this way; I mean, those are not even 20 lines of code. all good -just wanted to ask to get your why :)